pandas/io/formats/format.py

```diff
@@ -133,6 +133,8 @@
             Display DataFrame dimensions (number of rows by number of columns).
         decimal : str, default '.'
             Character recognized as decimal separator, e.g. ',' in Europe.
+        ending_header : bool, optional
+            Display header at the end of DataFrame.
```

Comment:
Can you mention what the default behaviour is?

Reply:
Done